packages/visualizations/src/components/MapPoi/types.ts

```diff
@@ -0,0 +1,37 @@
+import type { CircleLayerSpecification, StyleSpecification } from 'maplibre-gl';
+import type { BBox, FeatureCollection } from 'geojson';
+import type { Color } from '../types';
+
+export interface PoiMapOptions {
+    /** Configuration of map style */
+    style?: StyleSpecification | string;
+    /** Configuration for the layers to display Pois */
+    layerParams?: LayerParams;
+    /** Maximum boundaries of the map, outside of which the user cannot zoom/move
+     * Also set the position of the map when rendering.
+     * If undefined, will default, in order to:
+     * - Fit the content if the source is GeoJSON
+     * - The world
```

Comment:
There is only one kind of source now

```suggestion
     * If undefined, the map will fit the content.
```

Reply:
👍